perfkitbenchmarker/package_managers.py

```diff
@@ -109,7 +116,18 @@ def Startup(self):
 
   def InstallEpelRepo(self):
     """Installs the Extra Packages for Enterprise Linux repository."""
-    self.InstallPackages('epel-release')
+    try:
+      self.InstallPackages('epel-release')
+    except errors.VmUtil.SshConnectionError as e:
+      stdout, _ = self.RemoteCommand('cat /etc/redhat-release')
+      major_version = int(re.search('[0-9]', stdout).group(0))
+      if major_version == 6:
+        self.RemoteCommand('sudo rpm -ivh --force %s' % EPEL6_RPM)
```

Comment:
Nit: this would be cleaner if you set a local `rpm` based on `major_version`, then had a single `sudo rpm install` command.